Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityControllerMappingProfileInspector.cs

```diff
@@ -546,12 +549,23 @@ private void RenderInteractionList(SerializedProperty list, bool useCustomIntera
                     }
                     else
                     {
-                        if ((AxisType)axisType.intValue == AxisType.DualAxis)
+                        if ((AxisType)axisType.intValue == AxisType.SingleAxis)
                         {
                             EditorGUIUtility.labelWidth = customLabelWidth - 14f;
                             EditorGUIUtility.fieldWidth = 8f;
 
-                            EditorGUILayout.PropertyField(invertYAxis, InvertYAxisContent, GUILayout.Width(customLabelWidth));
+                            EditorGUILayout.PropertyField(invertXAxis, InvertXAxis, GUILayout.Width(customLabelWidth));
+
+                            EditorGUIUtility.labelWidth = defaultLabelWidth;
+                            EditorGUIUtility.fieldWidth = defaultFieldWidth;
+                        } else if ((AxisType)axisType.intValue == AxisType.DualAxis)
+                        {
+                            EditorGUIUtility.labelWidth = customLabelWidth * 0.5f;
+                            EditorGUIUtility.fieldWidth = 8f;
+                            
+                            InvertAxesEnum invertAxesEnum = InvertAxesEnum.Nothing;
+
+                            var modifedValue = (InvertAxesEnum)EditorGUILayout.EnumFlagsField(InvertAxes, invertAxesEnum, GUILayout.Width(customLabelWidth));
```

Comment:
It doesn't look like this value gets used to write the settings back to the properties anywhere. I've opened a PR with some fixes, assuming I'm not misunderstanding how this is being used.